vms/platformvm/txs/codec.go

```diff
@@ -97,3 +97,13 @@ func RegisterUnsignedTxsTypes(targetCodec linearcodec.Codec) error {
 	)
 	return errs.Err
 }
+
+func RegisterDUnsignedTxsTypes(targetCodec linearcodec.Codec) error {
+	errs := wrappers.Errs{}
+
+	errs.Add(
+		targetCodec.RegisterType(&TransferSubnetOwnershipTx{}),
```

Comment:
q: we add this tx to block codec but not tx codec (as we did for other transactions)? Why?

Reply:
Went with SkipRegistrations for now. Will keep this comment open as a reminder